notebooks/11-Advanced-Pandas.ipynb

```diff
@@ -2383,7 +2384,7 @@
     "tags": []
    },
    "source": [
-    "For this we'll use a sample of the Complete Journey data provided externally by 84.51:"
+    "For this we'll use a sample of the Complete Journey data provided externally by 84.51˚:"
```

Comment:
Do we have a link to the data? Might be useful if they want to explore more outside the class.

Reply:
added in d29c05a